src/server/api/schema.js

```diff
@@ -3879,14 +3879,24 @@ const rootResolvers = {
         return getStepsToUpdate(trx, findAndReplace);
       });
 
-      return steps.map((step) => {
-        return {
-          id: step.id,
+      const { searchString } = findAndReplace;
+
+      const stepsToChange = steps.flatMap((step) => {
+        const scriptOptions = step.script_options.filter((option) =>
+          option.includes(searchString)
+        );
+
+        // IDs get index added to it to not have multiple of the same IDs
+        // causes an issue with GraphQL is IDs are the same
```

Comment:
Nitpick for comment:
```suggestion
        // causes an issue with GraphQL if IDs are the same
```